relayer/listener.go

```diff
@@ -464,47 +363,40 @@ func (lstnr *Listener) getApplicationRelayer(
 	return nil
 }
 
-// Helper type and function to extract the information needed to relay a message
-// From the raw log information
-type parsedMessageInfo struct {
-	unsignedMessage    *avalancheWarp.UnsignedMessage
-	messageManager     messages.MessageManager
-	applicationRelayer *applicationRelayer
-}
-
-func (lstnr *Listener) parseMessage(warpLogInfo *relayerTypes.WarpLogInfo) (
-	*parsedMessageInfo,
+// TODONOW: Does this function make sense? There's probably a better way to organize this logic.
```

Comment:
Aside from the TODONOW, could you add a comment here describing what `GetAppRelayerMessageHandler` is meant to be used for? 🙏